speedtest/update.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
-current="$(curl -sSL 'https://pypi.python.org/pypi/speedtest-cli/json' | awk -F '"' '$2 == "version" { print $4 }')"
+current="$(curl -sSL 'https://pypi.python.org/pypi/speedtest-cli/json' | jq .info.version | tr -d '"')"
```

Comment:
Nice!  Instead of piping to `tr -d '"'`, let's just add `-r` to `jq` ("raw" output mode):

```suggestion
current="$(curl -fsSL 'https://pypi.python.org/pypi/speedtest-cli/json' | jq -r '.info.version')"
```